React/Views/RCTWebView.m

```diff
@@ -248,6 +270,13 @@ - (void)webView:(__unused UIWebView *)webView didFailLoadWithError:(NSError *)er
 
 - (void)webViewDidFinishLoad:(UIWebView *)webView
 {
+  if (_messagingEnabled) {
+    NSString *source = [NSString stringWithFormat:
+      @"window.postMessage = function(message) {"
+      "  window.location = '%@://%@?' + encodeURIComponent(JSON.stringify(message));"
+      "};", RCTJSNavigationScheme, RCTJSPostMessageHost];
```

Comment:
I added docs about this setting a global—if you don’t set an `onMessage` property in the webview, it won’t set a global.